lua/wikis/commons/Standings/Tiebreaker/Factory.lua

```diff
@@ -12,15 +12,20 @@ local String = Lua.import('Module:StringUtils')
 local TiebreakerFactory = {}
 
 local NAME_TO_CLASS = {
+	buchholz = 'Buchholz',
 	manual = 'Manual',
 	points = 'Points',
 	matchdiff = 'Match/Diff',
-	buchholz = 'Buchholz',
+	matchesplayed = 'Match/Count',
+	matcheswon = 'Match/Wins',
+	matchesdrawn = 'Match/Draws',
+	matcheswinpercent = 'Match/WinRate',
```

Comment:
Shouldn't these follow the same style? So IMO either `matchesdiff` or the others as `matchwins` etc.